### ISSUE TITLE
Recommended autoformatters?

### ISSUE THREAD

**Issue by cxong** [realpython/python-guide#931]

**Comment by dbader** [comment]
@cxong Agreed, would be great to expand this section. Personally I'm a big fan of `black`, I think it's awesome. If you want to put together a pull-request that expands the section that'd be fantastic!

**Comment by cxong** [comment]
I've only used black a little, so I don't think I have enough experience to author this section. Happy for someone else to take it up in the meantime.

**Comment by schilli91** [comment]
@dbader I'm thinking about adding a PR for this. Do you think this is still relevant? I'd add a sub section about auto-formatting giving brief instructions on how to install and use autopep8, yapf, and black.

**Comment by dbader** [comment]
@schilli91 Yes I think that'd be beautiful!

**Comment by schilli91** [comment]
@dbader I just opened the PR #1081. I'm happy to receive some feedback.
